### PR TITLE
Fix: update landing page for 2022

### PR DESCRIPTION
**Pull request by ckingbailey**
Update landing page with 2022 info

I had to add conditionals to handle case when total_contributions is 0

### Previews
Screenshot 2022-05-10 at 23-14-49 Open Disclosure Oakland

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,10 +2,12 @@
 layout: default
 header: Track the money in Oakland elections
 ---
+{% assign primary_date = '2022-06-07' %}
+{% assign general_date = '2022-11-08' %}
 
-{% assign march_totals = site.data.elections.oakland['2020-03-03'] %}
```
Comment (mikeubell) [inline index.html:6]: Shouldn't this use primary_date?

File: index.html [hunk 2]
```diff
@@ -2,10 +2,12 @@
...
-{% assign oakland_march_total_funds = march_totals.total_contributions %}
-{% assign oakland_march_residents_percentage = march_totals.total_contributions_by_source['Within Oakland'] | divided_by: oakland_march_total_funds %}
+{% assign primary_totals = site.data.elections.oakland['2020-03-03'] %}
```
Comment (mikeubell) [inline index.html]: ditto

File: index.html [hunk 3]
```diff
@@ -35,7 +37,7 @@ <h2 class="hero__table__header">Oakland Election Live Snapshot</h2>
             <thead class="hero__table__head">
                 <tr>
                     <th></th>
-                    <th class="hero__column__header">MARCH 2020</th>
+                    <th class="hero__column__header">{{ primary_date | date: "%B %Y" | upcase }}</th>
                     <th class="hero__column__header">NOVEMBER 2020</th>
```
Comment (mikeubell) [inline index.html]: This should use election_date

File: index.html [hunk 4]
```diff
@@ -2,10 +2,12 @@
...
 ---
+{% assign primary_date = '2022-06-07' %}
+{% assign general_date = '2022-11-08' %}
 
-{% assign march_totals = site.data.elections.oakland['2020-03-03'] %}
-{% assign oakland_march_total_funds = march_totals.total_contributions %}
-{% assign oakland_march_residents_percentage = march_totals.total_contributions_by_source['Within Oakland'] | divided_by: oakland_march_total_funds %}
+{% assign primary_totals = site.data.elections.oakland['2020-03-03'] %}
+{% assign oakland_march_total_funds = primary_totals.total_contributions %}
+{% assign oakland_march_residents_percentage = primary_totals.total_contributions_by_source['Within Oakland'] | divided_by: oakland_march_total_funds %}
```
Comment (mikeubell) [inline index.html]: Maybe changes these to oakland_primary_*

File: index.html [hunk 5]
```diff
@@ -53,7 +55,7 @@ <h2 class="hero__table__header">Oakland Election Live Snapshot</h2>
                 <tr>
                     <td class="hero__row__header">Top 3 third-party spenders (independent expenditures)</td>
                     <td class="hero__top-spenders">
-                      {% if march_totals.largest_independent_expenditures[0] != null %}
+                      {% if primary_totals.largest_independent_expenditures[0] != null %}
                       <ul>
                         {% for expenditure in site.data.totals.oakland-march-2020.largest_independent_expenditures %}
```
Comment (mikeubell) [inline index.html:69]: change to oakland-primary

**Review by mikeubell — COMMENTED** [review]
There are a number of additional places that still say oakland_march rather than oakland_primary

File: index.html (hunk 3 above)
Comment (mikeubell) [reply index.html]: In your image in the pull request it shows NOVEMBER 2022, so maybe you did not commit all your changes?

File: index.html (hunk 1 above)
Comment (ckingbailey) [reply index.html:6]: These are deletions

File: index.html (hunk 5 above)
Comment (ckingbailey) [reply index.html]: ✅ fixed

File: index.html (hunk 1 above)
Comment (mikeubell) [reply index.html:6]: Doh!

File: index.html (hunk 3 above)
Comment (mikeubell) [reply index.html]: Why is this still NOVEMBER 2020?

Comment (mikeubell) [reply index.html]: ok I see this  is changed in a later commit.

**Pull request merged**